websites/S/Stoodi/presence.ts

```diff
@@ -0,0 +1,131 @@
+const presence = new Presence({
+		clientId: "1043638037042712637",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000),
+	strings = presence.getStrings({
+		play: "presence.playback.playing",
+		pause: "presence.playback.paused",
+	});
+enum Assets {
+	play = "https://i.imgur.com/o4FaGfB.png",
```

Comment:
Enum's generally follow the Title case standard.

play -> Play
pause -> Pause